1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,30 @@
-function setAlarm() {}
+let remainingTime;
+let intervalId;
+
+function setAlarm() {
+  let setTimeValue = document.getElementById("alarmSet").value;
+  remainingTime = setTimeValue;
+  let minutes = Math.floor(remainingTime / 60);
+  let seconds = remainingTime % 60;
+
+  document.getElementById("timeRemaining").innerHTML = `Time Remaining: ${
+    minutes < 10 ? "0" : ""
+  }:${seconds < 10 ? "0" : ""}${seconds}`;
+
+  intervalId = setInterval(() => {
+    if (remainingTime > 0) {
+      remainingTime--;
+      let minutes = Math.floor(remainingTime / 60);
+      let seconds = remainingTime % 60;
+      document.getElementById("timeRemaining").innerText = `Time Remaining: ${
+        minutes < 10 ? "0" : ""
+      } : ${seconds < 10 ? "0" : ""}${seconds}`;
```

Comment:
Good way of representing the time remaining. You just forgot the minutes

`Time Remaining: ${
        minutes < 10 ? "0" : ""
      } **${minutes}**: ${seconds < 10 ? "0" : ""}${seconds}`;

Reply:
Ok,  thank you, Lorena, I'll fix it now.